metricbeat/docs/modules/zookeeper.asciidoc

```diff
@@ -6,7 +6,7 @@ This file is generated! See scripts/docs_collector.py
 == ZooKeeper module
 
 The ZooKeeper module fetches statistics from the ZooKeeper service. The default
-metricset is `mntr`.
+metricset is `mntr`. You can also activate `server` metricset.
```

Comment:
Based on the code, `server` is also a default. I like that so it's on by default.